pkg/ccl/backupccl/datadriven_test.go

```diff
@@ -87,6 +87,10 @@ type sqlDBKey struct {
 }
 
 type datadrivenTestState struct {
+	// clusters maps a name to its cluster
+	clusters map[string]serverutils.TestClusterInterface
```

Comment:
I wonder if we ought to change the `new-server` cmd to `new-cluster`, as it's really spinning up a new cluster and pushing all sql cmds to get served by the first server in the test cluster.

Reply:
sgtm 👍

Reply:
just pushed an extra commit for this.